src/System.Management.Automation/engine/remoting/fanin/OutOfProcTransportManager.cs

```diff
@@ -1548,7 +1559,12 @@ private static string ReadError(StreamReader reader)
             // Blocking read from StdError stream
             string error = reader.ReadLine();
 
-            if (string.IsNullOrEmpty(error) ||
+            if (error == null)
+            {
+                return error;
```

Comment:
No special reason except that I think of ReadError() helper method as a wrapper to StreamReader.  But I agree that it would be cleaner to just throw in ReadError()